test/unit_test/character_sheet_unit_tests/SpeedAndDefenseSupportTests.java

```diff
@@ -15,14 +15,14 @@
 	public void SetDexterity3GetSpeed12() {
 		CharacterSheet characterSheet = new CharacterSheet("CharacterSheet");
 		characterSheet.setData(Fields.DEXTERITY_VALUE, 3);
-		assertEquals(12, (int) characterSheet.getData(Fields.SPEED));
+		assertEquals(13, (int) characterSheet.getData(Fields.SPEED));

```

Comment:
Itt nem arról volt szó végül, hogy minden teszt elején állítod az armor-t ROBE-ra, hogy ne kelljen megbuzerálni a defaultjainkat? Én még arra is emlékszem hogy beszéltük hogy ahelyett hogy ```CharacterSheet characterSheet = new CharacterSheet("CharacterSheet");``` csinálhatsz egy privát függvényt ami tartalmazza ezt is meg az armor beállítást is és visszaköpi az 'előkészített' karakterlapot tesztelésre

Illetve légyszi a tesztfüggvények nevét is korrigáld